native/textern.py

```diff
@@ -24,6 +24,8 @@ def __init__(self):
             tmpdir_parent = os.path.join(
                 os.environ['XDG_RUNTIME_DIR'], 'textern')
             os.makedirs(tmpdir_parent)
+        except FileExistsError:
```

Comment:
There's also `exist_ok = True` we could pass to `makedirs()`, though this works for me too.